audio/libsndfile/Portfile

```diff
@@ -1,47 +1,55 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
-PortGroup           cmake 1.0
 PortGroup           github 1.0
 
 github.setup        libsndfile libsndfile 1.0.31
-revision            0
+revision            1
+
 categories          audio
 maintainers         nomaintainer
 platforms           darwin
 
 description         read and write files containing sampled sound
 long_description    \
-	libsndfile is a C library for reading and writing files	\
-	containing sampled sound through one standard library interface. \
-	libsndfile has the following main features: \
-	ability to read and write a large number of file formats, \
-	a simple, elegant and easy to use API, \
-	and on the fly format conversion.
+    libsndfile is a C library for reading and writing files \
+    containing sampled sound through one standard library interface. \
+    libsndfile has the following main features: \
+    ability to read and write a large number of file formats, \
+    a simple, elegant and easy to use API, \
+    and on the fly format conversion.
 license             LGPL-2.1+
+homepage            https://libsndfile.github.io/libsndfile/
+
 checksums           \
-      rmd160  8663673c3f8eaf463c7b499c732c53c9bba03c04 \
-      sha256  f082718e84a27d99c0e349151344b71dc2032e7aa2d799af0da48a8a270aa06a \
-      size    662676
-depends_build-append  port:pkgconfig
-depends_lib           port:flac port:libogg port:libvorbis
+    rmd160  ae3fc5bbcb10a034f3edc1240acacd9f1ec349a7 \
+    sha256  a8cfb1c09ea6e90eff4ca87322d4168cdbe5035cb48717b40bf77e751cc02163 \
+    size    875335
+
+github.tarball_from releases
+master_sites        https://github.com/libsndfile/libsndfile/releases/download/${version}
+distfiles           libsndfile-${version}.tar.bz2
 
-patchfiles            patch-CMakeLists.txt.diff
+depends_build       port:pkgconfig
+depends_lib         port:flac port:libogg port:libvorbis port:libopus
 
-configure.args-append -DBUILD_REGTEST=OFF \
-                      -DBUILD_SHARED_LIBS=ON \
-                      -DENABLE_EXTERNAL_LIBS=ON \
-                      -DENABLE_EXPERIMENTAL=OFF
+configure.args \
+    --disable-alsa \
+    --disable-octave \
+    --disable-silent-rules \
+    --disable-sqlite \
+    --disable-test-coverage \
+    --enable-external-libs
 
 variant no_external_libs description {Disable FLAC, Ogg and Vorbis support} {
     depends_build-delete  port:pkgconfig
-    depends_lib-delete    port:flac port:libogg port:libvorbis
-    configure.args-append -DENABLE_EXTERNAL_LIBS=OFF
-    configure.args-delete -DENABLE_EXTERNAL_LIBS=ON
+    depends_lib-delete    port:flac port:libogg port:libvorbis port:libopus
+    configure.args-append --disable-external-libs
+    configure.args-delete --enable-external-libs
```

Comment:
please use `configure.args-replace` instead

Reply:
This is literally verbatim what the Portfile from 2019 did. Have the standards changed since then? I was not aware there was such a thing as args-replace.

Reply:
Fixed, pushing.